pykechain/client.py

```diff
@@ -1048,12 +1048,14 @@ def create_activity(
             parent: Union[Activity, Text],
             name: Text,
             activity_type: ActivityType = ActivityType.TASK,
+            ref: Optional[Text] = None,
```

Comment:
I think this should not be None as default, but already the name slugified (same way as KE-chain does it)

Reply:
Not setting the ref will cause it be set by KE-chain, so this is intended. Also, if anyone consciously sets the ref, it is checked for validity with the slugify function, to make sure the expected ref is the resulting ref. Any specific change you want to see?

Reply:
Ok, then it is fine, wasn't sure how KE-chain will tackle it. The intended behaviour is indeed to have a ref at any time.